website/client/components/guilds/discovery.vue

```diff
@@ -0,0 +1,101 @@
+<template lang="pug">
+.row
+  sidebar(@:search="updateSearch", @:filter="updateFilters")
+
+  .col-10
+    .row
```

Comment:
just a note, not important: I don't think two rows or two col-12 are necessary here, the contente should just go under the header with this

```
.col-md-12
    .clearfix
       h1.float-left...
       b-form-select.floatright
    public-guild-item(...)